lib/src/conflicts.rs

```diff
@@ -68,23 +68,30 @@ impl<T> Conflict<T> {
         &self.adds
     }
 
-    /// Remove pairs of entries that match in the removes and adds.
+    /// Simplify the conflict by joining diffs like A->B and B->C into A->C. The
+    /// first add is considered a diff from the non-existent state.
```

Comment:
Perhaps mention that this also eliminates `A->A` diffs (as far as I can tell).

I also thought of this elaboration of the last sentence: "When interpreted as a series of diffs, the conflict's (i+1)-st add is matched with the i-th remove. The first add is considered a diff from the non-existent state.".

Reply:
Good idea. I moved the bit about how conflicts can be interpreted as a series of diffs to `struct Conflict`.